osal/src/os/posix/inc/os-freertos.h

```diff
@@ -40,21 +40,16 @@
 #include <stdlib.h>
 #include <string.h>
 #include <unistd.h>
```

Comment:
Posix dependency should be removed

Reply:
These are also in the rtems/vxworks builds but again no harm in removing them. But we are likely to use them in that file since every version of it has these files.